Error building networktables with cmake

**Describe the bug**
When attempting to install only networktables using cmake on a coprocessor, I ran into the following error when trying to build:

    pathToRepo/allwpilib/wpinet/src/main/native/cpp/UDPClient.cpp: In member function 'int wpi::UDPClient::set_timeout(double)`:
    pathToRepo/allwpilib/wpinet/src/main/native/cpp/UDPClient.cpp:240:18: error: aggregate 'wpi::UDPClient::set_timeout(double)::timeval tv' has incomplete type and cannot be defined
      240 |    struct timeval tv;
          |                   ^˜

**To Reproduce**
Steps to reproduce the behavior:
1. Clone WPI Lib repository
2. Make a build directory
3. Run: `cmake ../allwpilib/ -DWITH_JAVA=OFF -DWITH_JAVA_SOURCE=OFF -DWITH_CSCORE=OFF -DWITH_WPIMATH=OFF -DWITH_WPIUNITS=OFF -DWITH_EXAMPLES=OFF -DWITH_TESTS=OFF -DWITH_GUI=OFF -DWITH_SIMULATION_MODULES=OFF -DWITH_WPILIB=OFF`
4. Run `make -j4`
5. Comment out most of Stacktrace.cpp because alpine doesn't have execinfo
6. Re-run `make -j4`
7. Get the error shown above


 - Link to code: N/A

**Expected behavior**
`make` to build networktables

**Desktop (please complete the following information):**
 - OS: Alpine Linux

**Additional context**
On a coprocessor, a [Beelink Mini S12](https://www.bee-link.com/beelink-mini-s12-n95-mini-pc-clone-1) to be exact

--- a/wpinet/src/main/native/cpp/UDPClient.cpp
+++ b/wpinet/src/main/native/cpp/UDPClient.cpp
@@ -12,6 +12,7 @@
 #include <arpa/inet.h>
 #include <fcntl.h>
 #include <netinet/in.h>
+#include <sys/time.h>
 #include <unistd.h>
 #endif
 

